core/src/main/resources/jenkins/model/MasterBuildConfiguration/config.jelly

```diff
@@ -0,0 +1,30 @@
+<!--
+The MIT License
+Copyright (c) 2025, Stefan Spieker
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry title="${%# of executors}" field="numExecutors">
+        <f:number min="0" clazz="non-negative-number-required"/>
+    </f:entry>
+    <f:entry title="${%Labels}" field="labelString">
+        <f:textbox/>
+    </f:entry>
+    <f:slave-mode name="builtin.mode" node="app" />
```

Comment:
This cannot work. `node` isn't taking a string. I expect displaying the current mode doesn't work correctly in the current state of the PR.

Reply:
Ok, there is also a test failing, so I can verify if it is working. Can you help me how to do this correctly?

Reply:
`node="${app}"`

Reply:
Thanks a lot @mawinter69

Reply:
I verified in the debug environment that it works after the changes.